src/handbook/company/security/third-party-risk-management.md

```diff
@@ -0,0 +1,59 @@
+---
+navTitle: Third-Party Risk Management Policy
+---
+
+# Access Control Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @ZJvandeWeg    | 2023-06-01     |
+
+## Purpose
+
+To ensure protection of the organization's data and assets that are shared with,
+accessible to, or managed by suppliers, including external parties or third-party
+organizations such as service providers, vendors, and customers, and to maintain
+an agreed level of information security and service delivery in line with supplier
+agreements.
+
+## Scope 
+
+All data and information systems owned or used by FlowForge that are business
+critical and/or process, store, or transmit Confidential data.
```

Comment:
As it's capitalised, let's also link to the classification document. There's legal significance to the capital letter